react/OrderSeller.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { useRuntime } from 'vtex.render-runtime'
+
+import { useOrder } from './components/OrderContext'
+
+const OrderDate: FC = () => {
```

Comment:
Component/ file name mismatch, is this intended?

Reply:
That's an actual error. Thanks!